lib/logger.js

```diff
@@ -0,0 +1,3 @@
+var logger = require('@google/cloud-diagnostics-common').logger;
+var logLevel = process.env.GCLOUD_ERRORS_LOGLEVEL;
```

Comment:
As written, I think we disable all logging if the env. var. is not set. This would be a major regression. This needs to be fixed before #51 (or #51 needs to be merged back into this issue).